spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -313,6 +313,13 @@ contract PerpetualTranche is
     /// @notice Updates the reference to the rollover vault.
     /// @param newVault The address of the new vault.
     function updateVault(address newVault) public onlyOwner {
+        // If set to a non-zero address, we expect vault to be valid and implement `getTVL()`
+        if (newVault != address(0)) {
```

Comment:
As discussed offline, we probably don't need this condition since it's called by the owner.

Reply:
updated